recipes/backward-cpp/all/conanfile.py

```diff
@@ -81,7 +81,7 @@ def requirements(self):
             if self._has_stack_walking("libunwind"):
                 self.requires("libunwind/1.7.2", transitive_headers=True)
             if self._has_stack_details("dwarf"):
-                self.requires("libdwarf/20191104", transitive_headers=True, transitive_libs=True)
+                self.requires("libdwarf/0.9.1", transitive_headers=True, transitive_libs=True)
                 self.requires("libelf/0.8.13")
```

Comment:
```suggestion
                self.requires("libelf/0.8.13", transitive_headers=True)
```
https://github.com/bombela/backward-cpp/blob/v1.6/backward.hpp#L249

Reply:
Thanks @ericLemanissier, I wasn't sure how to go about resolving :)